.evergreen/config.yml

```diff
@@ -772,3 +795,9 @@ buildvariants:
   display_name: "${versions}/${php-versions}/${os-php7} — ${storage-engine}"
   tasks:
      - name: "test-standalone"
+
+- matrix_name: "libmongoc-versions-php7"
+  matrix_spec: {"os-php7": "debian92-test", "versions": "4.2", "php-versions": "7.2", "libmongoc-version": "*"}
+  display_name: "${versions}/${php-versions}/${os-php7} — ${libmongoc-version}"
```

Comment:
Does `${libmongoc-version}` just pull in the `id` from the `libmongoc-version` axis element? If so, that may only print "1.15-latest" or "master". Would it make sense to prefix that with "libmongoc: " for display purposes?

This is less of an issue with other display names (e.g. storage engines), so those IDs are pretty self-explanatory.

Reply:
If I'm not mistaken, this pulls the display name from the axis. Prefixing with "libmongoc" makes sense.